packages/snap-preact-components/src/components/Organisms/Results/Results.tsx

```diff
@@ -151,13 +152,15 @@ export const Results = observer((properties: ResultsProps): JSX.Element => {
 									return <ResultLayout controller={controller} result={result} layout={resultLayout} />;
 								} else {
 									return (
-										<Result
-											key={(result as Product).id}
-											{...subProps.result}
-											result={result as Product}
-											layout={props.layout}
-											controller={controller}
-										/>
+										<SearchResultTracker result={result} controller={controller as SearchController}>
```

Comment:
I'm thinking that we should only use the tracker when the controller exists. This way we can ensure that tracking only happens when coupled with a controller. Additionally, this will allow us to "switch" out the tracker (or pass it different props) based on the controller type in the future.